src/internal/config/config.go

```diff
@@ -183,15 +189,16 @@ func (cs *Service) GetCredentials() Credentials {
 }
 
 // SetCredentials allows to safely set and persist configuration settings.
-func (cs *Service) SetCredentials(accessToken, refreshToken string, expirationInSeconds int) error {
+func (cs *Service) SetCredentials(accessToken, refreshToken string, accessTokenExpiresAt, refreshTokenExpiresAt time.Time) error {
```

Comment:
Once we don't have any logic in a setter now, you can modify a method signature to accept `Credentials` instead.